index.html

```diff
@@ -0,0 +1,183 @@
+    </div>
+     <div class="info">
+        <h6 class="date">22.05.2022(THUR)~23(FRI)</h6>
+        <p class="venue">@ Church Auditorium, Lagos.</p>
+    </div>
+ </div>
+
+ <section class="prog-card">
+    <h5 class="h5">Main program</h5>
+    <hr class="hr2">
+    <div class="card-section">
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+                    <li><img class="img-w" src="./images/lecture.png" alt="lecture"></li>
+                   <li class="lpd" >Word</li>
+                 <li  class="msg">Out pouring of the spirit of God through His word by the man of God as He give utterances.</li>
+           </ul>
+        </div>
+</div>
+        
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd2">Prayers</li>
+            <li><p class="msg1">Its a time to draw down the presence of God and seeking His presence through prayers.</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div  class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd3">Worship</li>
+            <li><p class="msg2">praise and worhip secton to bring down the glory of God with annoited gospel musician.
+            </p>
+        </li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
+            <li class="lpd5">Discusion</li>
+            <li><p class="msg3">Building strong and vibrant youth for christ through moral building and responsible youth in the society</p></li>
+        </ul>
+    </div>
+</div>
+
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+            <li><img class="img-w"   src="./images/lecture.png" alt="#"></li>
+            <li class="lpd4">Fasting</li>
+            <li><p class="msg4">This event will take place on the last day of worship.</p></li>
+        </ul>
+    </div>
+</div>
+    </div>
+    <div class="join">
+    <p class="us">Worship With Us</p>
+</div>
+
+<div class="join2">
+    <p><a href=""> See More Program</a></p>
+</div>
+</section>
+
+<section class="ministers">
+
+     <h6 class="speakers">Featured Speakers</h6>
+    <hr class="hr1">
+    <div  class="minister-card"  id="js-card">
+    
+ <button type="button" class="more">more<i class="arrow down"></i></button>
+    </div>
+
+</section>
+    </main>
+
+    <footer class="footer1">
+      
+        <h2 class="fg">Partner</h2>
+         <hr class="hr-f">
+        <div class="grid"> 
```

Comment:
- [ ] These elements are not aligned properly, as the last reviewer requested.  Look:

![image](https://user-images.githubusercontent.com/57421823/164948830-019399f0-afd7-4d34-8057-555bded897e8.png)

Please, align them to have a little more separation like on the design, and also center them properly.

TIP: I achieved this by setting a `gap: 5%`:

![image](https://user-images.githubusercontent.com/57421823/164948880-1a8ac133-0e09-40a8-adde-a8f53a07c49a.png)

That way, your elements will be well separated, and more centered.

Reply:
done